source/manual/fall-back-to-aws-cloudfront.html.md

```diff
@@ -23,49 +23,78 @@ This backup CDN is currently provided by AWS CloudFront.
+
+```bash
+  # www-cdn.production.govuk.service.gov.uk
+  gds aws govuk-production-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='www.gov.uk'].DomainName | [0]"
+  # assets.publishing.service.gov.uk
+  gds aws govuk-production-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='assets.publishing.service.gov.uk'].DomainName | [0]"
+```
+
+### Staging
+
+For **staging DNS records**, open the following three pages as separate tabs:
+
+- [GCP Cloud DNS www.staging.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/www.staging.publishing.service.gov.uk./CNAME/edit-standard?project=govuk-production)
+- [GCP Cloud DNS assets.staging.publishing.service.gov.uk](https://console.cloud.google.com/net-services/dns/zones/publishing-service-gov-uk/rrsets/assets.staging.publishing.service.gov.uk./CNAME/edit-standard?project=govuk-production)
+- [AWS Route 53 publishing.service.gov.uk](https://us-east-1.console.aws.amazon.com/route53/v2/hostedzones#ListRecordSets/Z3SBFBO09PD5HF)
+
+You are going to update the `CNAME` records for two different domains, in both GCP and AWS:
+
+For **staging**, these two domains are `www.staging.publishing.service.gov.uk` and `assets.staging.publishing.service.gov.uk`
+You can get the `CNAME`s to use for the secondary CDN from the AWS CLI:
+
+```bash
+  # www.staging.publishing.service.gov.uk
+  gds aws govuk-staging-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='www.staging.publishing.service.gov.uk'].DomainName | [0]"
+  # assets.staging.publishing.service.gov.uk
+  gds aws govuk-staging-readonly aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Aliases.Items[0]=='assets.staging.publishing.service.gov.uk'].DomainName | [0]"
+```
+
+### Next steps
+
+In either case, the records should look like `d0000000000000.cloudfront.net.` (with 0s replaced with letters and numbers). Now you can manually update the `CNAME` records for both domains in both GCP and AWS, via the tabs you opened in your web browser earlier:
+
+- Change the canonical name from `www-gov-uk.map.fastly.net.` to the CloudFront domain name you found before, including the trailing period (e.g. `d0000000000000.cloudfront.net.`)
+- Test if the new Cloudfront domain is serving assets correctly, for example:
+
+```bash
+curl -vs https://assets.staging.publishing.service.gov.uk/media/662a74aa45f183ec818a72c2/dvsa-earned-recognition-vehicle-operators-accredited-list.csv/preview | grep cloudfront
+```
+
+- Perform a manual failover, for example by temporarily changing the new canonical domain name with a random prefix.
 - **On production only**, after performing the manual failover, you should also update our infrastructure-as-code to match the changes you just made:
```

Comment:
Should this section be in the `Production` section?

Reply:
Do you mean move it to that section, or duplicate it? I'm unclear. @kentsanggds

Reply:
Actually, I'm not sure we need to implement your comment: this point is in the "Next Steps" section which applies to both Staging and Production. @kentsanggds

Reply:
I think that section needs to be moved as it only applies to production. 
I don't think for Staging its necessary to update the terraform as part of the drill, unless we want to test the changes on Staging first before applying to Production if there is a real need to switch CDN?